pages/content/amp-dev/documentation/components/reference/amp-accordion@ar.md

```diff
@@ -0,0 +1,145 @@
+---
+$category@: layout
+formats:
+  - websites
+  - email
+  - ads
+teaser:
+  text: >-
+    Provides a way for viewers to have a glance at the outline of the content
```

Comment:
Some of these teaser texts seem to be translated and some not. Is that how they came in?

Reply:
Yes, they came in that way.